exercises/concept/guessing-game/.docs/introduction.md.tpl

```diff
@@ -0,0 +1,3 @@
+# Introduction
```

Comment:
I see that this will require contributors to know to use the tool to generate. That is unfortunate. That is why I am advocating for symlinking. But I will evaluate this thought another day. It does not need to be handled in this pull request.